clients/drcachesim/common/trace_entry.h

```diff
@@ -272,9 +272,17 @@ typedef enum {
     TRACE_MARKER_TYPE_CPU_ID,
 
     /**
-     * The marker value contains the function id defined by the user in the
-     * -record_function (and -record_heap_value if -record_heap is specified)
-     * option.
+     * The marker value contains the function id for functions traced by the user via
+     * the -record_function (and -record_heap_value if -record_heap is specified)
+     * option.  The mapping from this numeric ID to a library-qualified symbolic name
+     * is recorded during tracing in a file "funclist.log" whose format is described by
+     * the drmemtrace_get_funclist_path() function's documentation.
+     *
+     * This marker is also used to record parameter values for certain system calls such
+     * as for #OFFLINE_FILE_TYPE_KERNEL_SCHED.  These use large identifiers equal to
+     * #TRACE_FUNC_ID_SYSCALL_BASE plus the system call number (or its bottom 16 bits
+     * for 32-bit marker values).  These identifiers are not stored in the function
```

Comment:
Does this mean that for 32-bit marker values its simply bottom 16 bits without consideration to TRACE_FUNC_ID_SYSCALL_BASE? If so, let's move this part out of parenthesis. If not, maybe it is simpler to just say  TRACE_FUNC_ID_SYSCALL_BASE here and not the bits at all (since the user should rely on TRACE_FUNC_ID_SYSCALL_BASE only).

Reply:
It wouldn't really make sense to ignore the base and have just the bottom 16 bits as it just said it's a large identifier.  But I reworded to "#TRACE_FUNC_ID_SYSCALL_BASE plus the system call number (for 32-bit marker values
     * just the bottom 16 bits of the system call number are added to the base)."